kotlin-js/src/jsMain/kotlin/js/objects/Object.kt

```diff
@@ -19,7 +20,8 @@ external class Object internal constructor() {
         fun <T : Any> entries(o: ReadonlyRecord<String, T>): ReadonlyArray<JsTuple2<String, T>>
         fun entries(o: Any): ReadonlyArray<JsTuple2<String, Any?>>
         fun <R, T : R> freeze(o: T): R
-        fun <T : Any> fromEntries(entries: ReadonlyArray<JsTuple2<String, T>>): Record<String, T>
+        fun <T> fromEntries(entries: ReadonlyArray<JsTuple2<String, T>>): ReadonlyRecord<String, T>
+        fun <T> fromEntries(entries: JsIterable<JsTuple2<String, T>>): ReadonlyRecord<String, T>
```

Comment:
Signature with `JsIterable` - main (`Array: JsIterable`).
Could you move it please on first place?

Reply:
JFYI - `JsIterable` - parent type for `Array`.
Signature with `ReadonlyArray` requied only because parent type missed in Kotlin